website/docs/configuration/transient.mdx

```diff
@@ -123,11 +122,62 @@ Invoke Oh My Posh in `~/.config/fish/config.fish` and add the following line bel
 enable_poshtransientprompt
 ```
 
-Restart your shell or reload fish using `exec fish` for the changes to take effect.
+Restart your shell or reload `~/.config/fish/config.fish` using `exec fish` for the changes to take effect.
 
 </TabItem>
 </Tabs>
 
+## Disable the feature
+
+<Tabs
+  defaultValue="pwsh"
+  groupId="shell"
+  values={[
+    { label: 'powershell', value: 'pwsh', },
+    { label: 'cmd', value: 'cmd', },
+    { label: 'zsh', value: 'zsh', },
+    { label: 'fish', value: 'fish', },
+  ]
+}>
+<TabItem value="pwsh">
+
+You can use this function to disable the feature when you are in a PowerShell session:
+
+```powershell
+Disable-PoshTransientPrompt
+```
+
+</TabItem>
+<TabItem value="cmd">
+
+You cannot disable the feature when you are in a CMD session. You can **remove** the line below in the `oh-my-posh.lua` script to disable it permanently:
+
+```shell
+clink set prompt.transient always
```

Comment:
Just a note: this is a not a Lua statement. This command needs to be run in the shell session - it's a CMD command. And `clink set prompt.transient off` will turn transience off permanently. I think this is needs to be changed in the docs too - https://ohmyposh.dev/docs/configuration/transient

Reply:
😅 Oops, I missed this. Thank you for the good catch.